docs/deployment/universal-profile.md

```diff
@@ -130,11 +169,13 @@ Javascript's `File` object is only available client-side. If using LSPFactory in
 <script/>
 ```
 
-LSPFactory will create 5 resized versions of the passed image, with max sizes of 1800x1800, 1024x1024, 640x640, 320x320, 180x180. These resized images will be uploaded to IPFS and the metadata set in the LSP3 Profile.
+LSPFactory will create 5 resized versions of the passed image, with max sizes of 1800x1800, 1024x1024, 640x640, 320x320, 180x180. These resized images will be set inside the LSP3 metadata and attached to the `ERC725Account`.
+
+#### Using Image Buffer
 
-### Using Image Buffer
+If using LSPFactory in a Node environment where Javascript `File` object is not available, `profileImage` and `backgroundImage` can be uploaded by passing a File Buffer directly.
 
-If using LSPFactory in a Node environment where Javascript `File` object is not available, `profileImage` and `backgroundImage` can be uploaded by passing a File Buffer directly. The image mimetype must also be passed so that the Image can be reconstructed and resized. Supported mimetypes can be easily accessed by using the `SupportedImageBufferFormats` enum.
+The image mimetype must also be passed so that the image can be reconstructed and resized. Supported mimetypes can be easily accessed by using the `SupportedImageBufferFormats` enum exported by LSPFactory. Supported mime types are `image/png`, `image/bmp`, `image/jpeg` and `image/gif`.
```

Comment:
Suggestion:

Change from this ->  `Supported mimetypes can be easily accessed by using the `SupportedImageBufferFormats` ...`

To this -> `Supported mimetypes can be easily accessed via the `SupportedImageBufferFormats` ...`